src/common/ast-path.js

```diff
@@ -7,8 +7,8 @@ class AstPath {
   }
 
   get key() {
-    const { stack, siblings } = this;
-    return stack[stack.length - (siblings === null ? 4 : 2)] ?? null;
```

Comment:
The overall style of this file is to prefer micro-optimizations where possible, so let's stay consistent and not change this.